src/fragments/lib-v1/ssr/nextjs/getting-started.mdx

```diff
@@ -0,0 +1,338 @@
+## Prerequisites
+
+If you have not, please follow the [create application](/lib-v1/project-setup/create-application/q/platform/js/) guide to create your
+Next.js project, and Amplify backend.
+
+If you already have an existing Amplify backend, you can run the following command
+at the root of you project to integrate the Amplify backend.
+
+```bash
+amplify pull
+```
+
+## Configuration
+
+### Configure Amplify for Server Side Usage
+
+To enable Amplify library APIs usage on the server side, you will need to update
+your `next.config.js`.
+
+Import the Amplify configuration object from the `amplifyconfiguration.json` file
+that is generated by Amplify CLI, and use the `withAmplify` util functions to
+update the configuration of your Next.js app.
+
+For example, the `next.config.js` may contain the following content.
+
+```javascript
+const { withAmplify } = require('@aws-amplify/adapter-nextjs/with-amplify');
+const config = require('./src/amplifyconfiguration.json');
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // your Next.js app configuration
+}
+
+module.exports = withAmplify(nextConfig, config);
+```
+
+### Configure Amplify for Client Side Usage
+
+When you use the Amplify library on the client-side of your Next.js app, you will
+need to configure Amplify by calling the Amplify.configure as you would use Amplify
+in a client single-page application.
+
+<Callout>
+
+**NOTE:** To use the Amplify library on the client side in a Next.js app, you will
+need to pass the `ssr` parameter with a true value to the call of `Amplify.configure`.
+
+</Callout>
+
+```typescript
+'use client';
+
+import config from '@/aws-exports';
+import { Amplify } from 'aws-amplify';
+
+Amplify.configure(
+  config, {
+    ssr: true, // required when using Amplify with Next.js
+  }
+);
+
+export default function RootLayoutThatConfiguresAmplifyOnTheClient({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return children;
+}
+```
+
+<Callout>
+
+**TIP:** To avoid repetitive calls to `Amplify.configure`, you can call it once
+in a top-level client-side rendered layout component.
+
+</Callout>
+
+## Authentication
+
+You can use the Amplify Auth category APIs or the Amplify UI
+[Authenticator](https://ui.docs.amplify.aws/react/connected-components/authenticator)
+component to sign up and sign in your end users on the client side. With setting
```

Comment:
The Authenticator isn't yet compatible with v6, should remove references to the UI throughout preview docs.

```suggestion
You can use the Amplify Auth category APIs to sign up and sign in your end users on the client side. By setting
```